usage.txt

```diff
@@ -34,6 +34,11 @@ Environment variables:
         common-4.14
         common-4.9
         common-4.4
+  OBJSIZE:
+      If no OBJSIZE value is specified, the script will attempt to set OBJSIZE
+      to llvm-OBJSIZE-10 llvm-OBJSIZE-9, llvm-OBJSIZE-8, llvm-OBJSIZE-7,
```

Comment:
I think you meant for `llvm-size` here instead of `llvm-OBJSIZE`? Same with `${CROSS_COMPILE}size` instead of `${CROSS_COMPILE}OBJSIZE`?

Reply:
done